### ISSUE TITLE
Is there any provision for training models with gradient accumulation?

### ISSUE THREAD

**Issue by aman0044** [IDEACVR/MaskDINO#107]
Support for Gradient accumulation for lower batch size to accommodate large size images in single 16gb GPU?

**Comment by FengLi-ust** [comment]
Sorry, we did implement that. You are welcome to open a PR if you have any ideas.
